### ISSUE TITLE
The function fail if the user is not a member of any group

### ISSUE THREAD

**Issue by it-praktyk** [it-praktyk/Get-ADGroupDifferences#1]
```
Compare-Object : Cannot bind argument to parameter 'DifferenceObject' because it is null.
At \\domain.name\username\Data\foldername\Get-ADGroupDifferences.ps1:137 char:97
+ ... fferenceObject $CurrentUserGroups -IncludeEqual:$IncludeEqual)
+                    ~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidData: (:) [Compare-Object], ParameterBindingValidationException
    + FullyQualifiedErrorId : ParameterArgumentValidationErrorNullNotAllowed,Microsoft.PowerShell.Commands.CompareObje
   ctCommand 
```

**Comment by it-praktyk** [comment]
If the user is only the member of Domain users group - as is after the user creation - this group is assigned a primary group. The primary group is not returned by Get-ADUser as a part of MemberOf

More information here:
http://www.thomas-franke.net/about-active-directory-primary-group/

**Comment by it-praktyk** [comment]
Issue resolved by 3790bc3129f2a4bec9e7da99c4450775c79f4bf0
